Lib/distutils/util.py

```diff
@@ -45,8 +46,8 @@ def get_host_platform():
         return sys.platform
 
     # Set for cross builds explicitly
-    if "_PYTHON_HOST_PLATFORM" in os.environ:
-        return os.environ["_PYTHON_HOST_PLATFORM"]
+    if cross_compiling:
```

Comment:
Should not that be the first test, even before the  if `os.name=="nt"` which could return early ?

Reply:
If this is an issue then the problem exists in the current code so please enter a new issue. My understanding is that cross compilation has been implemented only for unix platforms.

Reply:
Just asking because in def get_platform(): you put that test at the top and it seems more logical, so yeah maybe it could be an already existing issue here that may show *if* cross compilation is implemented on os.name == 'nt'.